knowledge_runtime/knowledge_runtime/services/admin_executor.py

```
@@ -0,0 +1,207 @@
+            request.retriever_config
+        )
+
+        logger.info("Testing storage backend connection")
+
+        try:
+            success = storage_backend.test_connection()
+            return {
+                "success": success,
+                "message": "Connection successful" if success else "Connection failed",
+            }
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)